src/extras.jl

```diff
@@ -142,36 +152,70 @@ function cut(x::AbstractArray{T, N}, breaks::AbstractVector;
 
     n = length(breaks)
     if labels isa Function
-        @static if VERSION >= v"0.7.0-DEV.4524"
-            from = map(x -> sprint(show, x, context=:compact=>true), breaks[1:n-1])
-            to = map(x -> sprint(show, x, context=:compact=>true), breaks[2:n])
-        else
-            from = map(x -> sprint(showcompact, x), breaks[1:n-1])
-            to = map(x -> sprint(showcompact, x), breaks[2:n])
-        end
+        from = map(x -> sprint(show, x, context=:compact=>true), breaks[1:n-1])
```

Comment:
why do we pass strings not values to `levels` function?

Reply:
Woops. Will change that in another PR.